.ci/unit_test

```diff
@@ -48,7 +48,7 @@ test_with_coverage() {
 ################################################################################
 
 # To run a specific package, run TEST_PACKAGES=<PATH_TO_PACKAGE> make test
-TEST_PACKAGES="${TEST_PACKAGES:-"cmd pkg"}"
+TEST_PACKAGES="${TEST_PACKAGES:-"pkg/snapshot/snapshotter"}"
```

Comment:
> Why you are changing this ? Please revert this change

It was a temporary change to test only the snapshotter pkg in a not powerful machines like in concourse because those were needed for unit tests. 

I wanted to update this in the next commit along with few more changes. I will remove them in the next commit.